data/transition-sites/dclg_fire.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_fire
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/policies/fire-prevention-and-rescue
+tna_timestamp: 20120803062306
```

Comment:
This crawl [doesn't work](http://webarchive.nationalarchives.gov.uk/20120803062306/http://fire.gov.uk) - the latest working one is [`20100105005659`](http://webarchive.nationalarchives.gov.uk/20100105005659/http://fire.gov.uk), as @issyl0 said on the original ticket. The timestamp won't ever be used because of the global redirect, but it's still helpful to have it in Transition to refer to.